symfony/src/Controller/DeceasedReportsController.php

```diff
@@ -109,6 +111,30 @@ public function deceasedReportNew(Request $request, ParticipantSummaryService $p
         ]);
     }
 
+    /**
+     * @Route("/stats", name="deceased_report_stats")
+     */
+    public function getStats(SessionInterface $session, DeceasedReportsService $deceasedReportsService)
+    {
+        $organizationId = $session->get('siteOrganizationId');
+        $cache = new DatastoreAdapter(100);
+        $pendingReportCountCache = $cache->getItem('deceased_reports.pending.count');
```

Comment:
Shouldn't the cache key include the organization id?

Reply:
Uh, yes. 😬

Reply:
Addressed in 3d7a541